Makefile

```diff
@@ -1077,3 +1077,7 @@ help:
 	@echo ''
 
 endif #umask
+
+# should be last statement to be expanded immediatly, following should be only
+# deferred expansion of recipies and secondary expansion of dependecies
```

Comment:
Maybe say that you use this variable in order to detect later in write_deferred if we are called from a secondary expansion or not.